operations/node_management/Rebuild_NCNs.md

```diff
@@ -42,8 +42,14 @@ ncn# echo $XNAME
 
 Only follow the steps in the section for the node type that is being rebuilt:
 
-* [Worker node](Rebuild_NCNs/Prepare_Worker_Nodes.md)
-* [Master node](Rebuild_NCNs/Prepare_Master_Nodes.md)
+* [Worker node]
+```
+ncn-m001:# /usr/share/doc/csm/upgrade/1.2/scripts/rebuild/ncn-rebuild-worker-nodes.sh ncn-w001
```

Comment:
command prompt should be `ncn-m001`

Reply:
i am not sure i am following on this one